WordPress/src/main/res/values/strings.xml

```diff
@@ -263,6 +263,7 @@
     <!-- post view -->
     <string name="post_id">Post</string>
     <string name="upload_failed">Upload failed</string>
+    <string name="draft_uploaded">Draft uploaded</string>
```

Comment:
🤷‍♂️ 'Draft uploaded' seems to me to less clearly communicate that it's _still_ a draft, but now it's online.

> Still think this needs more thought?

I'm not sure, that's why I suggested getting some design input. If you prefer, we can add a note to visit this various language as part of our notifications work for async.